### PR TITLE
DB setup

### LINKED ISSUE #3: PostgreSQL Setup
Setup docker for PostgreSQL.

### LINKED ISSUE #4: Set up SpringBoot
Add spring boot dependency management and main class.

### LINKED ISSUE #8: Add CRUD Support
Tables need basic CRUD support to interface with them.

### PR DESCRIPTION
**Pull request by dantsar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,31 @@ This project intends to use the following frameworks and techonologies:
 # MORE TO COME
 
 # Build System
-To start we're going to use Gradle.
-To install Gradle on Linux (make sure that the gradle version is >6.0)
+You first need to create location to store database data
+```
+mkdir -p $HOME/srv/postgres
```
Comment (dantsar) [inline README.md:15]: I'd prefer the location of the DB to be in the project directory and not in $HOME. 
Maybe change it to something symbolic like <PROJ_HOME>

Can just update the .gitignore to actually ignore the location!

File: docker_build.sh [hunk 2]
```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```
Comment (dantsar) [inline docker_build.sh:1]: wonderful! going to docker_compose is the strat

File: Dockerfile [hunk 3]
```diff
@@ -1,19 +1,11 @@
...
-    && wget https://services.gradle.org/distributions/gradle-${VERSION}-bin.zip -P /tmp \
-    && unzip -d /opt/gradle /tmp/gradle-${VERSION}-bin.zip \
-    && ln -s /opt/gradle/gradle-${VERSION} /opt/gradle/latest \
-    && rm /tmp/gradle-${VERSION}-bin.zip
+FROM postgres
+#COPY *.sql /docker-entrypoint-initdb.d/
 
-ENV GRADLE_HOME=/opt/gradle/latest/bin/
-ENV PATH=$PATH:$GRADLE_HOME
+FROM maven:3.9.6-eclipse-temurin-21 AS build
```
Comment (dantsar) [inline Dockerfile:4]: <3

File: init_tables.sql [hunk 4]
```diff
@@ -20,8 +20,8 @@ CREATE TABLE playlist_tracks (
     playlist_id integer NOT NULL,
     song_id integer NOT NULL,
     PRIMARY KEY (id),
-    FOREIGN KEY (playlist_id) REFERENCES playlists(id),
-    FOREIGN KEY (song_id) REFERENCES tracks(id)
+    FOREIGN KEY (playlist_id) REFERENCES playlists(id)
+    --FOREIGN KEY (song_id) REFERENCES tracks(id) tracks doesnt exist
```
Comment (fred13kim) [inline init_tables.sql:26]: Not sure why you commented this line. You mention that tracks doesn't exist but there is a tracks table...

**Pull request merged**